### DISCUSSION TITLE
How can I check out code with "Enable local Actions only for this repository" permission

### DISCUSSION THREAD

**Discussion by lgylym** [community/community#27005] [Actions]
<p>When I set my repo’s permission to “Enable local Actions only for this repository”, I cannot use “actions/checkout@v1” to check out code anymore because it is in another repo. When I do that I get error message</p>
<p>Only actions in “xyz” are allowed for this repository</p>
<p>Is there some way for me to still be able to check out the code?</p>

**Comment by lgylym** [comment, marked as answer]
<p>I figured it out now. You need to fork the “actions/checkout” repo to your own organization and use that one instead.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/lgylym">@lgylym</a> Thanks a bunch for coming back and sharing your solution! </p>

**Comment by ChandSk** [comment]
<p>Hi,</p>
<p>what do you mean by the statement “You need to fork the “actions/checkout” repo to your own organization and use that one instead.” Can you please elaborate?</p>
